### ISSUE TITLE
Updating typeahead from maintained forks

### ISSUE THREAD

**Issue by janat08** [sergeyt/meteor-typeahead#160]
The original typeahead is no longer maintained, but there's maintained one here: https://github.com/corejavascript/typeahead.js
Would it be possible to merge their udpates?

**Comment by sergeyt** [comment]
@janat08 sure if someone makes a PR with this update, please feel free to be the first one :)

**Comment by janat08** [comment]
Lol, I just tried making a pull request from the maintained version, and it told me that they can't be compared, the idea then is to do this by hand?

**Comment by sergeyt** [comment]
yes you could just replace typeahead.bundle.js file, or consider to check getting dependency from npm. AFAIK meteor is capable to work with npm these days.

**Comment by janat08** [comment]
oh right.

**Comment by janat08** [comment]
Meh I spend several hours on this ruining meteor install along the way, to kind of give up after not being able to test it in my project with local packages folder.

**Comment by janat08** [comment]
Well I settled for just updating the bundle which lets me bind custom events which I think didn't happen previously. The sad thing that ele.focus() doesn't work still.
